### DISCUSSION TITLE
Why did autotrain change its ui and become less useable

### DISCUSSION THREAD

**Discussion by Sirdug** [huggingface/autotrain-advanced#609] [Q&A] [answered]
Whenever I see a video about autotrain from about 8 months to a year ago the UI looks super fancy and easy to use, I also notice there were more useful features to make it easier to use and an option to train more than one model. Is there any way to get that ui or version of autotrain for use today?
<img width="1142" alt="image" src="https://github.com/huggingface/autotrain-advanced/assets/80724707/8631311d-b795-45a9-8e7b-e8b6ea7c7e59">
<img width="1142" alt="image" src="https://github.com/huggingface/autotrain-advanced/assets/80724707/b2c0ef5e-2a7f-4af8-a900-2a1022687fe2">

**Comment by abhishekkrthakur** [comment, marked as answer]
Fancy is not always useful. The new autotrain is open source and has many tasks not available in the old version.
